src/ui.rs

```diff
@@ -151,6 +186,18 @@ impl Ui {
     pub fn flush(&mut self) -> io::Result<()> {
         match &mut self.output {
             UiOutput::Terminal { stdout, .. } => stdout.flush(),
+            // Unimplemented
```

Comment:
Do we need to do anything here? Or is this obsolete?

Reply:
AFAICT we only use this for the progress display, which won't apply to the pager. I don't know if there are plans to use `flush()` for output destined for the pager, but I suspect not.

Reply:
Hm, I could just call `.flush()` on the stdin, that probably does what we want.